### PR TITLE
conway: add tx body items

### PR DESCRIPTION
**Pull request by janmazak**

### CODE REVIEW

File: src/parsing/transaction.ts [hunk 1]
```diff
@@ -432,12 +602,26 @@ export function parseSignTransactionRequest(
...
+        tx.votingProcedures.every((voterVotes) => {
+          switch (voterVotes.voter.type) {
+            case VoterType.COMMITTEE_SCRIPT_HASH:
+            case VoterType.DREP_SCRIPT_HASH:
+              return true
+            default:
+              return false
+          }
+        }),
+        InvalidDataReason.SIGN_MODE_ORDINARY__VOTER_ONLY_AS_PATH,
```
Comment (davidmisiak) [inline src/parsing/transaction.ts:622]: incorrect error message

File: src/utils/parse.ts [hunk 2]
```diff
@@ -258,36 +259,54 @@ export function parseBIP32Path(
...
           errMsg,
         ),
       }
     default:
-      unreachable(stakeCredential)
+      unreachable(credential)
+  }
+}
+
+export function parseAnchor(params: AnchorParams): ParsedAnchor | null {
```
Comment (davidmisiak) [inline src/utils/parse.ts:295]: I find it a tiny bit confusing that `parseAnchor` is defined in `src/utils/parse.ts` (and not in `src/parsing/certificate.ts`), whereas `serializeAnchor` is in `src/interactions/serialization/txCertificate.ts` and not in `src/utils/serialize.ts`. (E.g. `parseCredential` and `serializeCredential` are consistent in this regard).

File: test/integration/__fixtures__/signTx.ts [hunk 3]
```diff
@@ -638,481 +643,924 @@ export const testsShelleyWithCertificates: SignTxTestCase[] = [
...
+            },
+          },
+        },
+      ],
+    },
     signingMode: TransactionSigningMode.ORDINARY_TRANSACTION,
+    additionalWitnessPaths: [],
     txBody:
-      'a500818258203b40265111d8bb3c3c608d95b3a0bf83461ace32d79336579a1939b3aad1c0b700018282583901eb0baa5e570cffbe2934db29df0b6a3d7c0430ee65d4c3a7ab2fefb91bc428e4720702ebd5dab4fb175324c192dc9bb76cc5da956e3c8dff821904d2a1581c95a292ffee938be03e9bae5657982a74e9014eb4960108c9e23a5b39a14874652474436f696e1a007838628258390114c16d7f43243bd81478e68b9db53a8528fd4fb1078d58d54a7f11241d227aefa4b773149170885aadba30aab3127cc611ddbc4999def61c1a006ca79302182a030a0807',
+      'a500818258203b40265111d8bb3c3c608d95b3a0bf83461ace32d79336579a1939b3aad1c0b700018182582b82d818582183581c9e1c71de652ec8b85fec296f0685ca3988781c94a2e1a5d89d92f45fa0001a0d0c25611a002dd2e802182a030a048183028200581c1d227aefa4b773149170885aadba30aab3127cc611ddbc4999def61c581cf61c42cbf7c8c53af3f520508212ad3e72f674f957fe23ff0acb4973',
```
Comment (davidmisiak) [inline test/integration/__fixtures__/signTx.ts:1068]: There seems to be something wrong here - this exact txBody CBOR is present for several different test cases (the same is true for other new txBodies).

Comment (janmazak) [reply test/integration/__fixtures__/signTx.ts]: Should be fixed now.

**Review by davidmisiak — APPROVED** [review]

**Pull request merged**